modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+// about/ossm-about-concepts-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-concepts-kiali_{context}"]
+= {KialiProduct}
+
+{KialiProduct} provides visibility into your service mesh by showing you the microservices in your service mesh, and how they are connected.
+
+Kiali provides observability into the Service Mesh running on OpenShift Container Platform. Kiali helps you define, validate, and observe your Istio service mesh. It helps you to understand the structure of your service mesh by inferring the topology, and also provides information about the health of your service mesh.
+
+Kiali provides an interactive graph view of your namespace in real time that provides visibility into features like circuit breakers, request rates, latency, and even graphs of traffic flows. Kiali offers insights about components at different levels, from Applications to Services and Workloads, and can display the interactions with contextual information and charts on the selected graph node or edge.
+
+Kiali also provides the ability to validate your Istio configurations, such as gateways, destination rules, virtual services, mesh policies, and more. Kiali provides detailed metrics, and a basic Grafana integration is available for advanced queries. Distributed tracing is provided by integrating {TempoName} and {OTELName} into the Kiali console.
+
+== {KialiProduct} architecture
+
+Kiali is based on the open source link:https://kiali.io/[Kiali project]. Kiali is composed of two components: the Kiali application and the Kiali console.
+
+Kiali Server (back end):: This component runs in the container application platform and communicates with the service mesh components, retrieves and processes data, and exposes this data to the console. The Kiali application does not need storage. When deploying the application to a cluster, configurations are set in ConfigMaps and secrets.
+
+Kiali console (front end):: The Kiali console is a web application. 	The Kiali application serves the Kiali console, which then queries the Kiali Server for data to present it to the user.
+
+In addition, Kiali depends on external services and components provided by the container application platform and Istio.
+
+Red Hat Service Mesh (Istio):: Istio is a Kiali requirement. Istio is the component that provides and controls the service mesh. Although Kiali and Istio can be installed separately, Kiali depends on Istio and will not work if it is not present. Kiali needs to retrieve Istio data and configurations, which are exposed through Prometheus and the cluster API.
```

Comment:
I'm not sure what we call the "OpenShift cluster's API" or the "Kubernetes API server". Just "cluster API" doesn't seem right.

Reply:
In OCP core, it's "Cluster APIs": https://docs.openshift.com/container-platform/4.17/rest_api/cluster_apis/cluster-apis-index.html

Reply:
Those are actually different. "Cluster APIs" are a specific resource of the OpenShift API Server, the one you linked, and Kiali does not use that resource. Kiali uses the OpenShift API server not the "Cluster APIs". This is why I think "Cluster API" is confusing.